### DISCUSSION TITLE
Screensaver toggler

### DISCUSSION THREAD

**Discussion by Absoluties** [rocksdanister/lively#937] [Ideas]
I really like the screensaver feature of Lively. However, I don't use it because since it activates after a period of inactivity, it activates when I'm still using my PC if the said period is low (for example when watching Youtube / Twitch and not touching the keyboard) or activates way too late.
That's why I would love if you could bind a key to activate screensaver (just like streamers can change stages on their stream with a button) or if the option was showing up when you right click the tray icon (along the options to close and pause wallpapers).
Hope you'll give it a go, I love your work ! Cheers !

**Comment by rocksdanister** [comment]
> I don't use it because since it activates after a period of inactivity, it activates when I'm still using my PC if the said period is low (for example when watching Youtube / Twitch and not touching the keyboard) or activates way too late.

If you turn off the in-app settings and follow this step instead then it won't happen:
https://github.com/rocksdanister/lively/wiki/Screen-Saver#2-windows-screensaver-file

>if you could bind a key to activate screensaver 

Screensavers can also be triggered using scripts:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#screen-saver

or key bind using AutoHotkey:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#screen-saver-1

**Reply by Absoluties** [reply]
Thanks for the reply, you solution is working like a charm ! Even though I'd still like to see the features I suggested implemented, your answer solved my problem.
